landingai/common.py

```diff
@@ -168,3 +168,22 @@ def decode_bitmap_rle(bitmap: str, encoding_map: Dict[str, int]) -> List[int]:
         map_number = encoding_map[map_letter]
         flat_mask.extend([int(map_number)] * int(num))
     return flat_mask
+
+
+def in_notebook() -> bool:
```

Comment:
same here, prefer moving it to a dedicated python module

Reply:
To be discussed